test/release/examples/users-guide/base/casting.chpl

```diff
@@ -0,0 +1,18 @@
+{
+  var myString = "1234";
+  var myInt: int;
+
+  myInt = myString: int;
+  myInt = myInt + 1;
```

Comment:
I intentionally avoided it just because I hadn't introduced += yet and have been trying to avoid relying on things until I introduce them as much as possible...  But I did go back and forth before landing here.